code/python/example.py

```diff
@@ -1,21 +1,24 @@
-# pip3 install neo4j-driver
-# python3 example.py
+# (requires Python 3.6 or newer and pip, check with `pip --version`)
+# pip install neo4j
+# python example.py
 
-from neo4j import GraphDatabase, basic_auth
+from neo4j import GraphDatabase
 
-driver = GraphDatabase.driver(
-  "neo4j+s://demo.neo4jlabs.com:7687",
-  auth=basic_auth("mUser", "s3cr3t"))
+def products_of_category(transaction, movie_title_):
+    cypher_query = (
+        "MATCH (m:Movie {title:$movieTitle})<-[:ACTED_IN]-(a:Person)" 
+        "RETURN a.name as actorName")
+    result = transaction.run(cypher_query, movie_title=movie_title_)
+    return result.value("actorName")
 
-cypher_query = '''
-MATCH (m:Movie {title:$movieTitle})<-[:ACTED_IN]-(a:Person) RETURN a.name as actorName
-'''
+uri = "neo4j+s://demo.neo4jlabs.com:7687"
+auth = ("mUser", "s3cr3t")
+driver = GraphDatabase.driver(uri, auth=auth)
```

Comment:
single line with inline values, no need for the extra vars

Reply:
I guess it has to do with line width?

Reply:
It is to do with line length.

Reply:
79 chars per line according to PEP8. I don't want us to look like we don't know our Python foo.
https://peps.python.org/pep-0008/#maximum-line-length